parm/FV3.input.yml

```diff
@@ -83,7 +78,7 @@ FV3_HRRR:
     <<: *RRFS_v1beta_phys
     cdmbgwd: [3.5, 1.0]
     do_mynnsfclay: True
-    do_sfcperts: !!python/none
+    do_sfcperts:
```

Comment:
The latest version of pyyaml did not need/want this tag.

Reply:
Does this represent a potential back-compatibility issue for any yaml config files? If so it should probably be documented.

Reply:
@gspetro-NOAA do you have a suggestion where this kind of information can/should go? Perhaps release notes would be sufficient, but in the absence of a release, do we have an ongoing list of changes since the release?

Reply:
I also got a nice off-line suggestion to use `null` as the value here instead of leaving it blank. It will be more explicit and not make unfamiliar users feel like it might have been a mistake or oversight.

Reply:
@christinaholtNOAA For what it's worth, issue #981 contains all of the PRs and their commit messages since the SRW v2.2.0 release.

Reply:
@christinaholtNOAA There's also a "[Known Issues](https://github.com/ufs-community/ufs-srweather-app/wiki/Known-Issues)" page for issues related to `develop` or general issues. Known issues related to a specific release can be added to the appropriate release page in the Known Issues section.